docs/workflows/verify-images.rst

```diff
@@ -0,0 +1,271 @@
+``remapIdentity`` and full registry path prefix needs to be specified.
+
+::
+
+        $ cat /etc/containers/policy.json
+        {
+          "default": [{"type": "reject"}],
+          "transports": {
+            "docker": {
+              "fluffy.example.com/local-ubi8-repo": [
+                {
+                  "type": "signedBy",
+                  "keyType": "GPGKeys",
+                  "keyPath": "/path-to-rh-key.txt",
+                  "signedIdentity": {
+                      "type": "remapIdentity",
+                      "prefix": "fluffy.example.com/local-ubi8-repo",
+                      "signedPrefix": "registry.access.redhat.com/ubi8/ubi-micro"
+                  }
+                }
+              ]
+            },
+            "containers-storage": {
+            "": [{"type": "insecureAcceptAnything"}] /* Allow copy operations on any images stored in containers storage (e.g. podman push) */
+            }
+          }
+        }
+
+
+       podman pull fluffy.example.com/local-ubi8-repo:8.5
+
+
+Verify images pushed into Pulp Container registry
+-------------------------------------------------
+
+Usecase: Pulp Container Registry serves container images that were pushed into it (signed or not).
+
+Push an image into Pulp Container registry and sign it in one go:
+
+::
+
+        $ podman push fluffy.example.com/myrepo/test-image:foo --sign-by pupsik@redhat.com
+        Copying blob 252fdf0c3b6a done  
+        Copying config 829374d342 done  
+        Writing manifest to image destination
+        Signing manifest
+        Storing signatures
+
+        $ podman pull fluffy.example.com/myrepo/test-image:foo 
+        Trying to pull fluffy.example.com/myrepo/test-image:foo...
+        Getting image source signatures
+        Checking if image destination supports signatures
+        Copying blob 58147e24f776 skipped: already exists  
+        Copying config 829374d342 done  
+        Writing manifest to image destination
+        Storing signatures
+        829374d342ae65a12f3a95911bc04a001894349f70783fda841b1a784008727d
+
+
+        $ cat  /etc/containers/policy.json
+        {
+          "default": [{"type": "reject"}],
+          "transports": {
+            "docker": {
+               "fluffy.example.com": [
+                {
+                  "type": "signedBy",
+                  "keyType": "GPGKeys",
+                  "keyPath": "/path-to-pupsik-key.gpg"
+                }
+              ]
+            },
+            "containers-storage": {
+            "": [{"type": "insecureAcceptAnything"}] /* Allow copy operations on any images stored in containers storage (e.g. podman push) */
+            }
+          }
+        }
+
+
+Verify images from multiple registries
+--------------------------------------
+
+Usecase: Pull and verify content from Pulp Container registry and also other registries.
+
+To pull and verify images coming directly from ``registry.access.redhat.com`` create a separate
```

Comment:
```suggestion
To pull and verify images coming also from ``registry.access.redhat.com`` create a separate
```

Just to emphasize that there is more than one registry. It says it in the title and the use case but the example is referring to one registry, and only policy lists multiple. Nitpick.